jdi-bdd-tests/src/main/java/io/github/com/custom/MenuItem.java

```diff
@@ -1,8 +1,8 @@
 package io.github.com.custom;
 
-import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.elements.common.uiElement;
 
-public class MenuItem extends UIElement {
+public class MenuItem extends uiElement {
```

Comment:
No need in this changes. Please see https://www.cs.rice.edu/~cork/book/node15.html#:~:text=By%20convention%2C%20Java%20programs%20are,are%20written%20entirely%20capital%20letters.

Reply:
fixed